### PR TITLE
[NIFI-9423-NIFI-9429]: Show icon and tooltip for Parameters with leading and/or trailing whitespace

### PR DESCRIPTION
**Pull request by mtien-apache**
Thank you for submitting a contribution to Apache NiFi.

Please provide a short description of the PR here:

#### Description of PR

_NIFI-9423 - Show icon and tooltip for Parameter values that have leading and/or trailing whitespaces_
_NIFI-9429 - Parameters should allow blank values that are non-null (only whitespace)_

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with **NIFI-XXXX** where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `main`)?

- [ ] Is your initial contribution a single, squashed commit? _Additional commits in response to PR reviewer feedback should be made on this branch and pushed to allow change tracking. Do not `squash` or use `--force` when pushing to allow for clean monitoring of changes._

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via `mvn -Pcontrib-check clean install` at the root `nifi` folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [x] Have you verified that the full build is successful on JDK 8?
- [ ] Have you verified that the full build is successful on JDK 11?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the `LICENSE` file, including the main `LICENSE` file under `nifi-assembly`?
- [ ] If applicable, have you updated the `NOTICE` file, including the main `NOTICE` file found under `nifi-assembly`?
- [ ] If adding new Properties, have you added `.displayName` in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check GitHub Actions CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by mtien-apache** [comment]
@mcgilman Thanks for starting a review! I've added more updates. Can you check again with the latest changes?

**Comment by mcgilman** [comment]
Thanks... reviewing...

**Comment by mtien-apache** [comment]
@mcgilman I updated `valueFormatter` to accommodate for a parameter value with multilines to always show an ellipsis on the first line.

I also included a fix for [NIFI-9459](https://issues.apache.org/jira/browse/NIFI-9459) (Empty string check does not disable value field on open). 

Can you check again? Thanks!

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/main.css [hunk 1]
```diff
@@ -59,10 +59,6 @@ div.context-menu-provenance {
     text-overflow: ellipsis;
 }
 
-.hide-multi-line {
-    max-height: 16px;
-}
-
```
Comment (mtien-apache) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/main.css]: Note: No longer needed now that we're using the `multi-line-clamp-ellipsis` class.

**Review by mcgilman — APPROVED** [review]
Thanks for the PR @mtien-apache and thanks for all the effort and subsequent commits. +1 will merge....

**Pull request merged**